packages/components/src/LoaderApp/constant.js

```diff
@@ -0,0 +1,80 @@
+const LOADER_STYLE = `@keyframes app-loader-spin {
+	0% {
+		transform: rotate(0deg);
+	}
+	100% {
+		transform: rotate(360deg);
+	}
+}
+
+@keyframes app-loader-fadeIn {
+	0%,
+	100% {
+		opacity: 0.5;
+	}
+	50% {
+		opacity: 1;
+	}
+}
+
+.tc-loader-app-container {
+	display: flex;
+	height: 100vh;
+	width: 100vw;
+	background: #f6f6f6;
```

Comment:
That color should be imported front our common SCSS variables.

Reply:
I can't import scss at this time we are not in the app, so outside of the bundle